Link Host certificates if available in Admin Container

sumukhballal opened the pull request:
<!--
Tips:
- Please read the CONTRIBUTING document to understand our process and our requests for PRs.
- Please file an issue before creating a PR so we can discuss the change and confirm it's not already being worked on.
-->
**Description of changes:**

This change links host certificates if it has been updated on the Bottlerocket host. It checks if the path `/.bottlerocket/certs` exists, if so links the certs to the ca-trust source.

 - [x] Test variant aws-k8s-1.28 boot
 - [x] Test variant aws-k8s-1.24 boot
 - [x] Container specific tests

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

jpculp commented on start_admin.sh:
```diff
@@ -35,6 +36,15 @@ declare -r SSHD_CONFIG_FILE="${SSHD_CONFIG_DIR}/sshd_config"
 declare -i available_ssh_methods=0
 
+# Link host certs if present into container & run update-ca-trust
+if [ -e "${HOST_CERTS}" ]; then
```
```suggestion
if [[ -d "${HOST_CERTS}" ]]; then
```

sumukhballal committed:
certs: link host certs if present

In certain cases, if host certificates are present at the path '/.bottlerocket/certs', we would link them to the
ca-trust and update-ca-trust process to pickup the new certificates.

sumukhballal commented:
Addressed @jpculp 's comments.
